cmd/bootstrap/cmd/partner_infos.go

```diff
@@ -0,0 +1,229 @@
+package cmd
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"github.com/onflow/cadence"
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go-sdk/client"
+	"github.com/onflow/flow-go/cmd"
+	"github.com/onflow/flow-go/cmd/util/cmd/common"
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/model/bootstrap"
+	"github.com/onflow/flow-go/model/encodable"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+const (
+	// Index of each field in the cadence NodeInfo as it corresponds to cadence.Struct.Fields
+	// fields not needed are left out.
+	idField = iota
+	roleField
+	networkingAddressField
+	networkingKeyField
+	stakingKeyField
+)
+
+const (
+	flowNodeAddrPart    = "nodes.onflow.org"
+	defaultPartnerStake = 100
```

Comment:
isn't the default partner stake `1000`?

Reply:
The [default stake we seem to have is 100](https://github.com/onflow/flow-go/blob/316c36815fdab8788a8fbf67e30a4e992ae5d434/cmd/bootstrap/cmd/genconfig.go#L65), but we are actually using 1000 (from looking at the latest root snapshot). I'd suggest we make this 1000, and also change the default in `genconfig` to 1000 to be consistent with our actual usage.